Exec/Make.PeleLMeX

```diff
@@ -54,6 +54,15 @@ ifeq ($(USE_SOOT), TRUE)
    DEFINES += -DNUM_SOOT_MOMENTS=$(NUM_SOOT_MOMENTS)
 endif
 
+ifeq ($(USE_PELERAD), TRUE)
+   $(info ******** Radiation model is activated ********)
+   DEFINES += -DPELELM_USE_RAD
+   ifeq ($(USE_HIP), TRUE)
+       $(info ******** Radiation module is using hip ********)
```

Comment:
Do we need this define? PELERAD_USE_HIP == USE_HIP, so why cary an extra flag?

Reply:
With the current clang version on Frontier, std::filesystem doesn't exist.
Instead, I have to do 

#ifdef PELERAD_USE_HIP
#include <experimental/filesystem>
#else
#include <filesystem>
#endif

The name may be confusing, this is considered a temporary fix to get cases running on Frontier. Once the clang compiler on Frontier moves std::experimental::filesystem to std::filesystem, the flag is no longer needed.